### DISCUSSION TITLE
withdraw problem!

### DISCUSSION THREAD

**Discussion by oldfoxx13** [Chia-Network/chia-blockchain#5816] [Support]
Hello. I have 2 coins in my wallet, but I can not withdraw them in any way - the system writes that full synchronization is required, but it is already complete! Rebooted and synced several times already! What to do?

**Comment by oldfoxx13** [comment]
Now I updated the program to the latest version (I thought it would help), and my coins completely disappeared from my wallet!!!! Although at the bottom you can see the history of their receipt to the account just an hour ago! Screenshots are attached!

**Reply by mmatrosov** [reply]
> Screenshots are attached!

Where?

**Reply by oldfoxx13** [reply]
how do I send them to you?

**Reply by oldfoxx13** [reply]
I sent it to your email address

**Comment by oldfoxx13** [comment]
Здравствуйте. По Вашему имени я понял, что могу писать по-русски, это так?
Во вложении скриншоты. Благодарю Вас за внимание, очень жду ответа!

сб, 22 мая 2021 г. в 23:40, Mikhail Matrosov ***@***.***>:

> Screenshots are attached!
>
> Where?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Chia-Network/chia-blockchain/discussions/5816#discussioncomment-772507>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUFYITWWGN72BCJRKPJLJXLTPAJFPANCNFSM45K4UYPQ>
> .
>


-- 

Андрей Нахшунов
коммерческий директор
ООО "Агробор"
+7 909 986 02 13

**Reply by mmatrosov** [reply]
К сожалению, я не могу помочь, я тут недавно. Я лишь заметил, что тому, кто мог бы помочь, нужно будет предоставить информацию. Чем больше информации - тем больше вероятность, что вам помогут.

**Comment by oldfoxx13** [comment]
Hello

**Comment by oldfoxx13** [comment]
Вы могли бы мне ответить?

**Comment by oldfoxx13** [comment]
Я послал вам скриншоты на почту, с которой вы написали.

**Comment by oldfoxx13** [comment]
Две входящие транзакции от 05/22/2021 19:14:29. А монет в кошельке 0!

**Comment by oldfoxx13** [comment]
Благодарю за ответ. Так что же мне теперь делать? К кому обращаться* Или
мне подождать пока вы передадите информацию кому-то, кто может помочь?

вс, 23 мая 2021 г. в 23:34, Mikhail Matrosov ***@***.***>:

> К сожалению, я не могу помочь, я тут недавно. Я лишь заметил, что тому,
> кто мог бы помочь, нужно будет предоставить информацию. Чем больше
> информации - тем больше вероятность, что вам помогут.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Chia-Network/chia-blockchain/discussions/5816#discussioncomment-774545>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUFYITQWOP3N2ZEQBRZW42LTPFRHBANCNFSM45K4UYPQ>
> .
>


-- 

Андрей Нахшунов
коммерческий директор
ООО "Агробор"
+7 909 986 02 13

**Reply by mmatrosov** [reply]
Боюсь, на меня рассчитывать не стоит, я тут просто мимо проходил ) Да и вообще тут никто никаких гарантий не даёт, всё довольно кустарно. Могу предложить попробовать создать новую ноду фул ноду со старым ключом. Но надо понять, как работает миграция кошелька. В этом я не разбирался.

**Comment by oldfoxx13** [comment]
Понял, спасибо в любом случае. Я так и не понял вообще как работает синхронизация кошелька на разных машинах... А можете посоветовать куда написать?

**Comment by BrandtH22** [comment]
@oldfoxx13 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
